main/src/ca/uwaterloo/flix/language/errors/RedundancyError.scala

```diff
@@ -338,6 +340,38 @@ object RedundancyError {
     })
   }
 
+  /**
+    * An error raised to indicate that a function expression is useless.
```

Comment:
If we want to have this error message, we should try to give some more information.